frontend/src/pages/Game.js

```
@@ -21,14 +22,26 @@ function Game() {
+
     // スコアを保存
-    axios.post('/api/game/progress', {
-      user_id: 1,
-      level: parseInt(level),
-      score: score
-    });
+    const token = localStorage.getItem('token');
+    if (token) {
+      axios.post('/api/game/progress', {
+        level: parseInt(level),
+        score: score
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)